examples/test_core_lib/core_lib/data_layers/service/user_service.py

```diff
@@ -1,23 +1,27 @@
+from core_lib.data_layers.data_access.db.crud.crud_soft_data_access import CRUDSoftDataAccess
 from core_lib.data_transform.result_to_dict import ResultToDict
 from core_lib.data_layers.service.service import Service
+from core_lib.error_handling.not_found_decorator import NotFoundErrorHandler
 from examples.test_core_lib.core_lib.data_layers.data_access.user_data_access import UserDataAccess
 
 
 class UserService(Service):
-    def __init__(self, data_access: UserDataAccess):
+    def __init__(self, data_access: UserDataAccess or CRUDSoftDataAccess):

```

Comment:
`UserDataAccess or CRUDSoftDataAccess` which one is it?

Reply:
if you see in the `examples/test_core_lib/core_lib/test_core_lib.py` file we have 2 `UserService` instances one is passing `UserDataAccess` whereas the other is passing `CrudUserDataAccess`, to handle this the following change was made

Reply:
1. use only UserDataAccess
2. create CustomerService that will accept CustomerDataAccess(CRUD)